sys/conf/kern.mk

```diff
@@ -336,6 +339,7 @@ LD_EMULATION_i386=elf_i386_fbsd
 LD_EMULATION_mips= elf32btsmip_fbsd
 LD_EMULATION_mipshf= elf32btsmip_fbsd
 LD_EMULATION_mips64= elf64btsmip_fbsd
+LD_EMULATION_mips64c128= elf64btsmip_cheri_fbsd
```

Comment:
Does it still need the `_cheri`? My LLD change should have made it unnecessary in general, not just RISC-V.

Reply:
Just checked, not needed anymore.